packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -0,0 +1,37 @@
+import {Component, Element, h} from '@stencil/core';
+import ArrowRightIcon from 'coveo-styleguide/resources/icons/svg/arrow-right-rounded.svg';
+import ArrowLeftIcon from 'coveo-styleguide/resources/icons/svg/arrow-left-rounded.svg';
+import {Button} from '../../common/button';
+
+@Component({
+  tag: 'atomic-segmented-facet-scrollable',
+  styleUrl: 'atomic-segmented-facet-scrollable.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacetScrollable {
+  @Element() private host!: HTMLElement;
+
+  render() {
+    return (
+      <div class="flex h-9">
+        <Button
+          style="square-neutral"
+          class="flex shrink-0 basis-8 justify-center items-center rounded"
+          ariaHidden="true"
+        >
+          <atomic-icon class="w-3.5" icon={ArrowLeftIcon}></atomic-icon>
+        </Button>
+        <div class="wrapper-segmented flex flex-row overflow-x-scroll">
```

Comment:
detail, doesn't really matter since you hide the scrollbar already
```suggestion
        <div class="wrapper-segmented flex flex-row overflow-x-auto">
```

Reply:
Messes with the overflow.
<img width="1725" alt="Screen Shot 2022-06-21 at 12 10 12 PM" src="https://user-images.githubusercontent.com/97299176/174847145-a37ad2de-4272-4f77-84e5-a6581cdb8327.png">